app/models/signon_identity.rb

```diff
@@ -79,13 +88,26 @@ def logout_url(redirect_to: nil)
     url.to_s
   end
 
+  def accessible_estates
+    @_ae ||= self.class.accessible_estates(@permissions).order(:nomis_id).to_a
+  end
+
+  def accessible_estate?(estate)
+    accessible_estates.include?(estate)
+  end
+
+  def default_estate
+    accessible_estates.first || fail('Should never be nil')
```

Comment:
This could be nil if SSO gets set up with a new organisation with access to PVB that has not yet been setup in PVB. With the current version if this happens the user will get a 500